lstchain/scripts/lstchain_merge_hdf5_files.py

```diff
@@ -76,7 +76,7 @@ def main():
 
     if args.noimage:
         keys = get_dataset_keys(file_list[0])
-        keys = [k for k in keys if 'image' not in k]
+        keys = {k for k in keys if 'image' not in k}
```

Comment:
Is this still needed given that it is later converted to `set()`?

Reply:
Not strictly needed, but if keys is expected to be a set why not make it a set directly?